src/applications/vaos/components/InfoAlert.jsx

```diff
@@ -19,6 +20,7 @@ import PropTypes from 'prop-types';
 export default function InfoAlert({
   backgroundOnly = false,
   children,
+  classes = '',
```

Comment:
I don't really like this pattern but since this component wraps a web component it doesn't register any `className`. This is needed to apply those classes inside the component reference instead.

Reply:
We could still call this prop `className`, since that's more what you'd expect from a React component.